service/src/main/java/bio/terra/externalcreds/controllers/SwaggerController.java

```diff
@@ -19,10 +20,10 @@
 
   public SwaggerController() {
     try {
-      InputStream swaggerClientIdStream =
+      @Cleanup
```

Comment:
I _think_ I need this, but it's hard to tell...

Reply:
I think this should just be
```
      try (val reader = new BufferedReader(new InputStreamReader(new ClassPathResource("rendered/swagger-client-id").getInputStream()))) {
        swaggerClientId = reader.readLine();
      }
```

The try syntax will automatically close the reader which closes all nested readers/streams. I think you should use just `readLine()` instead of reading the whole file into a single string… if there is more than 1 line something is probably wrong.